src/main/webapp/js/utils/table-utils.jsx

```diff
@@ -56,16 +59,51 @@ const getTableCell = (item, props) => {
   );
 };
 
+const getTableFilter = (filter, onChange, format, fieldType) => {
+  let Filter;
+  switch (fieldType) {
+    case ENUM:
+      Filter = EnumFilter;
+      break;
+    default:
+      Filter = TextFilter;
+  }
+
+  return (
+    <Filter filter={filter} onChange={onChange} format={format} />
+  );
+};
+
 const getTableColumn = (props) => {
   const {
-    name, displayName, filterable,
+    name, displayName, filterable, format, fieldType,
   } = props;
 
   return {
     Header: displayName,
     accessor: name,
     filterable,
     Cell: item => getTableCell(item, props),
+    filterMethod: (filter, row) => {
+      let filterValue = null;
+      if (filter.value === 'ALL') {
+        filterValue = true;
+      }
+      if (fieldType === 'ENUM' && format) {
+        _.forEach(format.split(','), (val) => {
```

Comment:
you don't need to use format here, you can just check the value:
`filterValue = (row[filter.id] === filter.value);`